cfep-01.md

```diff
@@ -0,0 +1,135 @@
+should be set to Draft.
+
+## Submission
+
+Once complete, this draft should be submitted as a pull request to the
+[conda-forge/conda-forge-enhancement-proposals](https://github.com/conda-forge/conda-forge-enhancement-proposals)
+repository with the Status changed to Proposed.  At this point, members of the
+conda-forge community will review the submission.
+
+## Review
+
+CFEP review will begin once a pull request has been made.  All members of the
+conda-forge community are welcome and encouraged to participate in the review
+of CFEPs in a civil and respectful manner.  The CFEP champion(s) should be
+prepared to answer questions on the proposal and make updates to the
+proposal as recommended by the community.
+
+## Resolution
+
+All CFEPs will be resolved as either *Rejected*, *Accepted* or *Deferred*
+depending on the consensus of the community. Once the community has reached a
+consensus, the CFEP champion should update the Status of proposal and add a
+link to the discussion to the table.  Regardless of the resolution, the pull
+request should be merged once these tasks have been done.
+
+## CFEP Maintenance
+
+For *Rejected* CFEPs, no modifications are made after the pull request is merged.
+*Accepted* CFEPs should be updated with a link to the pull request(s) of the
+implementation and the status changed to *Implemented* when the proposed
+changes have been implemented.  *Deferred* CFEPs can be re-submitted in a new
+pull request after an appropriate amount of time.
+
+# CFEP Content
+
+All CFEPs should begin with a top level table with the following information:
+
+<table>
+<tr><td> Title </td><td> A short title of the proposal </td>
+<tr><td> Status </td><td> Draft | Proposed | Accepted | Rejected | Deferred | Implemented </td></tr>
+<tr><td> Author(s) </td><td> Full Name &lt;full.name@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Aug 30, 2016</td></tr>
+<tr><td> Updated </td><td> Aug 30, 2016</td></tr>
+<tr><td> Discussion </td><td> link to the issue where the CFEP is being discussed, NA before submission </td></tr>
+<tr><td> Implementation </td><td> link to the PR for the implementation, NA if not available </td></tr>
+</table>
+
+This table should be followed by a **Abstract** section and then
+additional sections as needed by the proposal.  Some section that may be
+included if appropriate for the proposal include.
+
+    * Specification -- The technical details of the proposed change.
+    * Motivation -- Why the proposed change is needed.
+    * Rationale -- Why particular decisions were made in the proposal.
+    * Backwards Compatibility -- Will the proposed change break existing
+      packages or workflows.
+    * Alternatives -- Any alternatives considered during the design.
+    * Sample Implementation -- Links to prototype or a sample implementation of
+      the proposed change.
+    * FAQ -- Frequently asked questions (and answers to them).
+    * Reference -- Any references used in the design of the CFEP.
```

Comment:
One field that might be nice, but is not necessarily filled out by the author would be a Resolution field. Basically what did we do with this CFEP and why.

This goes beyond linking the PR(s) that implements this functionality, but includes why did we accept it or why did we reject. These summary opinions can be very helpful for historic record, proposing new CFEPs in the same vein, or generally getting a feel for what makes for a working CFEP. PEPs do this with a [link to a comment]( https://www.python.org/dev/peps/pep-0001/#pep-header-preamble ). Though they also use a mailing list. GitHub comments as they are with the ability to edit in a quasi-transparent manner may not be the best thing to link to. Instead maybe a couple line summary that goes in with the rejection would be nice.